ui/src/components/Register/Register.vue

```diff
@@ -91,10 +84,23 @@
           <!-- Formerly incarcerated-->
           <div class="register-info">
             <span>{{ felonText }} </span>
-            <cv-link :inline="true" :href="felonLink" target="_blank">{{
+            <cv-link style="margin-left: 0" :inline="true" :href="felonLink" target="_blank">{{
               $t('moreInformation')
             }}</cv-link>
           </div>
+
+          <!-- Register day of voting -->
+          <div class="register-faq-header">{{ $t('faq') }}</div>
+          <div class="register-faq" v-if="'election_day' in info.register">
+            <span>{{ $t('registerElectionDay') }}</span
```

Comment:
span last tag indicator split across lines...

Reply:
The files are formatted with the "prettier" package and I cannot find a setting for not splitting tags like that. I might try the `<!-- prettier-ignore -->` comment on that one to see if I can make it look better.